spikes/oms-log-analytics/README.md

```diff
@@ -4,59 +4,60 @@ Prometheus is a popular open source metric monitoring solution and is a part of
 
 Reference Documentation [here](https://docs.microsoft.com/en-us/azure/azure-monitor/containers/container-insights-prometheus-integration).
 
-## Steps to Scrape Metrics into Log Analytics
+## Installation on AKS Clusters
 
-1. Save the Workspace ID and the key for log Analytics work space from the Azure dashboard
-2. If deploying to a AKS cluster, check whether you already have OMS Agent running.
-   1. Check if `omsagent` Pod is running: `kubectl get pods -A | grep omsagent`
-   2. If `omsagent` is running then only apply the config file: `kubectl apply -f 3-container-azm-ms-agentconfig.yaml`
-   3. Skip to Step 6
-3. To use OMS Agent Daemonset and deployment:
-   1. Create the log analytics secrets with kubectl:
+For non-AKS clusters please see [non-aks/README.md](non-aks/README.md).
 
-    ```bash
+For installing on AKS clusters, we recommend enabling Azure Monitor for Containers through your Infrastructure as Code solution.
+Please see, [Enable Container Insights](https://docs.microsoft.com/en-us/azure/azure-monitor/containers/container-insights-onboard) for instructions on enabling Azure Monitor for Containers for your cluster.
```

Comment:
It might be good to include the one-liner CLI command they include on there:

```md
    az aks enable-addons -a monitoring -n $Ngsa_AKS_Name -g $Ngsa_App_RG
```

Otherwise, this is pretty straightforward to do following the link you included.

Reply:
I'm hesitant to add the specific instructions here, beyond the link to the official install docs. My thought is that install instructions will change over time and this way we won't have to maintain and update the instructions, rather the docs linked to will be updated for us